src/init/map.init.js

```diff
@@ -26,77 +24,45 @@ export const initMap = async (
 
     const spawnpointsCharacters = {};
 
-    for (const layer of layers) {
```

Comment:
making this change might be bad for performance because using array.forEach will always run through the entire array. I used for of here to early return which is not possible by using forEach